### PR TITLE
BTC Withdraw

### PR DESCRIPTION
**Pull request by aulneau**
This PR adds the ability for users to withdraw BTC from the Stacks wallet's fuel address.

### CODE REVIEW

File: app/common/lib/transactions.js [hunk 1]
```diff
@@ -42,6 +42,61 @@ export const ERRORS = {
...
+      btcAmount,
+      false,
+      true
+    );
+
+    const txBytes = estimateTXBytes(rawTx);
+
+    const feeRate = await config.network.getFeeRate();
+
+    const fee = feeRate * txBytes;
```
Comment (kyranjamie) [inline app/common/lib/transactions.js:88]: Noting the possibility for imprecision

File: app/store/configureStore.dev.js [hunk 2]
```diff
@@ -29,7 +28,7 @@ const configureStore = initialState => {
   // Router Middleware
   const router = routerMiddleware(history);
   middleware.push(router);
-  middleware.push(persistMiddleware());
+  // middleware.push(persistMiddleware());
```
Comment (kyranjamie) [inline app/store/configureStore.dev.js:31]: This can definitely be removed? 🤔

**Review by kyranjamie — APPROVED** [review]
Looks good @aulneau 

I'm glad I started working with React in the hooks era, not a fan of all the HOC connect wrappers

File: app/store/configureStore.dev.js (hunk 2 above)
Comment (yknl) [reply app/store/configureStore.dev.js]: fixed

**Pull request merged**